### DISCUSSION TITLE
Multiple Unique Faces?

### DISCUSSION THREAD

**Discussion by vararth** [Bing-su/adetailer#204] [Ideas]
Not sure how this would work - but when I use prompts to generate multiple people - I am making use of Textual Inversions most of the time. Likewise, ADetailer is PERFECT and I love the nice faces it makes! The issue is, when I have multiple people in my prompt - for example, a male and a female - and I am using a textual inversion for the male and female faces - there is no way to tell ADetailer which face is male and which face is female - and this causes the final faces to mix up a bit and look similar, so the male and female look identical. 
If there is any prompting trick that I could use - I'm all ears - otherwise do consider this as a very minor request to perhaps add a way to differentiate between multiple characters in a generation to use different prompts for their faces :)

**Comment by Bing-su** [comment]
Is this a problem that can't be solved with wildcards or [dynamic prompts](https://github.com/adieyal/sd-dynamic-prompts)? I think that's enough for diversity.

**Comment by vararth** [comment]
I haven't used dynamic prompts, can try that.
Unsure if it will solve the problem though because I am talking about different male and female face using textual inversion in the adetailer prompt - using female face ti will make the male look female, using a male ti will make the female look male ; using both male and female ti will make hybrid faces for both the male and female so faces are all detected faces use the same adetailer prompt for identical face info inpainting

**Reply by jmcq01** [reply]
Does this work for you? https://github.com/Bing-su/adetailer/wiki/Advanced

**Comment by JayarCurio** [comment]
Try going to the settings tab and setting "Sort Bounding boxes by" to "Left-to-Right." https://github.com/Bing-su/adetailer/wiki/Settings#sort-bounding-boxes-by

That will make it a little easier to predict which side of [SEP] goes to which character.
